daemon/src/taker_cfd.rs

```diff
@@ -93,19 +92,11 @@ impl Actor {
         self.current_maker_offers = takers_perspective_of_maker_offers.clone();
         tracing::trace!("new maker offers {:?}", takers_perspective_of_maker_offers);
 
-        // If there's no offer, it might as well be hard-coded to BTC
-        let symbol = self
-            .current_maker_offers
-            .clone()
-            .and_then(|offer| offer.long.map(|offer| offer.contract_symbol))
-            .unwrap_or(ContractSymbol::BtcUsd);
-
         if let Err(e) = self
             .projection_actor
-            .send(projection::Update((
-                symbol,
+            .send(projection::Update(
                 takers_perspective_of_maker_offers.clone(),
-            )))
+            ))
```

Comment:
👍 I think these simplifications of `taker_cfd` and `projection` are in line with what I've done in https://github.com/itchysats/itchysats/pull/2680. 

We'll have to adapt some code to merge them together, but I'm now sending lists of pure `model::Offer`s around, which is a step further than what you've done here.